src/constants/__init__.py

```diff
@@ -0,0 +1 @@
+MAX_ACTION_LENGTH = 50
```

Comment:
May be better renamed `MAX_ACTION_NAME_LENGTH` in case additional similar constants are added in the future (which I expect because of the addition of this file).